File: python3-9/1037.py
# -*- coding: utf-8 -*-

value = float(input())

if value < 0:
    print('Fora de intervalo')
elif value >= 0 and value <= 25:
    print(f'Intervalo [0,25]')
elif value <= 50:
    print(f'Intervalo (25,50]')
elif value <= 75:
    print(f'Intervalo (50,75]')
elif value <= 100:
    print(f'Intervalo (75,100]')
else:
    print('Fora de intervalo')

File: C++20/1037.cpp
#include <iostream>
using namespace std;
 
int main() {
    double x;

    cin >> x;

    if (x >= 0 and x <= 25) {
        cout << "Intervalo [0,25]" << "\n";
    } else if (x > 25 and x <= 50) {
        cout << "Intervalo (25,50]" << "\n";
    } else if (x > 50 and x <= 75) {
        cout << "Intervalo (50,75]" << "\n";
    } else if (x > 75 and x <= 100) {
        cout << "Intervalo (75,100]" << "\n";
    } else {
        cout << "Fora de intervalo" << "\n";
    }

    return 0;
}